src/publickey.js

```diff
@@ -365,8 +365,10 @@ PublicKey.prototype._getID = function _getID() {
  * @param {String|Network=} network - Which network should the address be for
  * @returns {Address} An address generated from the public key
  */
-PublicKey.prototype.toAddress = function(network) {
-  var Address = require('./address');
+PublicKey.prototype.toAddress = function (network) {
+  // #weirdstuff resolve circular dependency.
```

Comment:
I think we should leave this comment here because we wanted to come back to this later to resolve